diagon-alley/src/Components/HeroImage/HeroImage.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Button } from '..';
```

Comment:
I think button component not import properly

Reply:
Its imported from index of components and hereImage is also in components so its that way